--- bsddb3-3.4.0/setup.py.wiget	Mon Sep  9 12:35:35 2002
+++ bsddb3-3.4.0/setup.py	Mon Sep  9 12:39:15 2002
@@ -52,7 +52,7 @@
                     print "Found BerkeleyDB installation at " + instdir
                     break
 
-            incdir = os.path.join(dir, "include/db3")
+            incdir = os.path.join(dir, "include/db4")
             if os.path.exists(incdir):
                 libdir = os.path.join(dir, "lib")
                 print "Found db3 header files at " + incdir
@@ -68,8 +68,8 @@
     # figure out from the base setting where the lib and .h are
     if not incdir: incdir = os.path.join(BERKELEYDB_DIR, 'include')
     if not libdir: libdir = os.path.join(BERKELEYDB_DIR, 'lib')
-    if not '-ldb' in LIBS:
-        libname = ['db']
+    if not '-ldb4' in LIBS:
+        libname = ['db4']
     else:
         libname = []
     utils = []
